app/components/dashboard/collection_component.html.erb

```diff
@@ -29,7 +29,7 @@
               <% end %>
             <% end %>
           </td>
-          <td><%= work.state.humanize %></td>
+          <td><%= render Works::StateDisplayComponent.new(work: work) %></td>
```

Comment:
This is part of @mjgiarlo's efforts to normalize how we display state info.  He will have a PR forthcoming after _this_ PR is merged to finish this work.